src/app.js

```diff
@@ -1,25 +1,53 @@
 import React from 'react'
+import Navbar from './nav-bar'
+import hash from './hash'
+import MyFlashcards from './my-flashcards'
 import FlashcardForm from './flashcard-form'
 
 export default class App extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
+      view: { path: 'view' },
```

Comment:
You should initialize `state.view` using the `location.hash` so refreshing the page still keeps you on the same screen.